client/src/styles/HomePage.css

```diff
@@ -62,11 +62,13 @@
 .category-text {
   font-size: 16px;
   margin: 7px;
+  word-wrap: break-word;
 }
 
 .category-card {
+  min-width: 90px;
   width: 90px;
```

Comment:
min-width should actually be 100px so that categories of length 14 characters would be displayed on two lines. With 90px they are divided onto three lines.

I was also thinking that do we want the categories to have equal spacing of margins, because with `min-width: 100px` the spaces are quite big. Regarding the margins I experimented with something like this:

```
.category-card {
  margin: 0px 10px;
  width: 100px;
  height: 110px;
  text-align: center;
}

.category-card p {
  margin: 7px 0px
}
```

You can test this and decide which is best.

Reply:
This is a good idea. Added this to the home page since the category cards share the same CSS. This revealed an issue with the main page overflowing so fixed that as well